builtin/rebase--helper.c

```diff
@@ -90,8 +90,13 @@ int cmd_rebase__helper(int argc, const char **argv, const char *prefix)
 		return !!transform_todos(flags);
 	if (command == CHECK_TODO_LIST && argc == 1)
 		return !!check_todo_list();
-	if (command == SKIP_UNNECESSARY_PICKS && argc == 1)
-		return !!skip_unnecessary_picks();
+	if (command == SKIP_UNNECESSARY_PICKS && argc == 1) {
+		struct object_id oid;
+		ret = skip_unnecessary_picks(&oid);
```

Comment:
Maybe `ret` could be declared here, as it looks like it is used only in this if clause.

Reply:
I fixed this in my new pull request.